lib/versioning/node/index.spec.ts

```diff
@@ -10,22 +10,35 @@ describe('versioning/node/index', () => {
     DateTime.local = dtLocal;
   });
 
-  test.each`
-    currentValue | rangeStrategy | currentVersion | newVersion  | expected
-    ${'1.0.0'}   | ${'replace'}  | ${'1.0.0'}     | ${'v1.1.0'} | ${'1.1.0'}
-    ${'~8.0.0'}  | ${'replace'}  | ${'8.0.2'}     | ${'v8.2.0'} | ${'~8.2.0'}
-  `(
-    'getNewValue($currentValue, $rangeStrategy, $currentVersion, $newVersion, $expected) === $expected',
-    ({ currentValue, rangeStrategy, currentVersion, newVersion, expected }) => {
-      const res = nodever.getNewValue({
+  describe('getNewValue', () => {
```

Comment:
Do we really need all those extra `describe` secrtions? Can't we simply add that to the test itself to reduce nesting?

Reply:
Yes, we can. I just wanted to make it consistent. We had mixed approaches in this file. Will change everything so that we get a flat list 👍

Reply:
https://github.com/renovatebot/renovate/pull/13466/commits/10ee0f12c84affc44ec039ea57b6a3e29cad703c